addons/binding/org.openhab.binding.cbus/src/main/java/org/openhab/binding/cbus/internal/cgate/CGateConnectException.java

```diff
@@ -0,0 +1,30 @@
+/**
+ *  CGateInterface - A library to allow interaction with Clipsal C-Gate.
```

Comment:
Thanks @daveoxley for getting back to us!
I agree with you, we should not include your sources, but really only a binary - any "alterations/simplifications" should thus be contributed to your project as a PR and we would include a new version afterwards. ASL2 license is absolutely fine, we can include such a lib in openHAB easily. So it would be great if you could change (or add it as dual-licensing) the license and release a version of it - until then, we will have to wait with the further processing of this PR. Thanks!